pages/job-category/[JobCategory].js

```diff
@@ -0,0 +1,18 @@
+import { styled } from "styled-components";
+import { JobsDummyData } from "../api/db";
+import { useRouter } from "next/router";
+import JobList from "../components/JobList";
+
+export default function JobsCategory({ jobs }) {
+  const router = useRouter();
+  const jobCategory = router.query.JobCategory;
```

Comment:
Your code is not wrong, but it might be tidier to de-structure here. 

```suggestion
  const { jobCategory }  = router.query;
```

Reply:
Thanks for the feedback Pete! But I'm getting this message upon de-structuring the variable : "Cannot destructure property 'jobCategory' of 'router.query.JobCategory' as it is undefined."

Reply:
Ah... my mistake. `jobCategory` is not the same as `JobCategory `. 

Feel free to leave this as before.